docs/en/multi_modal/xcomposer2d5.md

```diff
@@ -29,7 +29,7 @@ from lmdeploy.vl.constants import IMAGE_TOKEN
 pipe = pipeline('internlm/internlm-xcomposer2d5-7b')
 
 image = load_image('https://raw.githubusercontent.com/open-mmlab/mmdeploy/main/tests/data/tiger.jpeg')
-response = pipe((f'describe this image {IMAGE_TOKEN}', image))
+response = pipe((f'{IMAGE_TOKEN} describe this image', image))
```

Comment:
motivation?

Reply:
<img width="1415" alt="image" src="https://github.com/user-attachments/assets/372f98ae-2a3c-4a09-a878-1100df7aad77">

Reply:
Is it better if we hide the IMAGE_TOKEN in the guide?